Packs/GitLab/Integrations/GitLab/.pack-ignore

```diff
@@ -0,0 +1,2 @@
+[file:GitLab_image.png]
+ignore=IM111
```

Comment:
Remove this. Let's fix the image, no need to ignore it. 
DM me for instructions